packages/compiler/src/server/types.ts

```diff
@@ -101,6 +101,9 @@ export interface Server {
   getInitProjectContext(): Promise<InitProjectContext>;
   validateInitProjectTemplate(param: { template: InitTemplate }): Promise<boolean>;
   initProject(param: { config: InitProjectConfig }): Promise<boolean>;
+
+  // Following custom capacities are added for supporting update imports on file rename from IDE (vscode for now) so that IDE can trigger compiler
+  updateImportsOnFileRename(param: { oldFilePath: string; newFilePath: string }): Promise<string[]>;
```

Comment:
can't we using the watchedFilesChanged() in lsp to monitor the rename event so that everything can be done in lsp?